webrtc/RTCRtpTransceiver-stop.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCRtpTransceiver.prototype.stop</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+async function negotiate(offerer, answerer) {
+  const offer = await offerer.createOffer();
+  await Promise.all([
+    offerer.setLocalDescription(offer),
+    answerer.setRemoteDescription(offer)
+  ]);
+  const answer = await answerer.createAnswer();
+  await Promise.all([
+    answerer.setLocalDescription(answer),
+    offerer.setRemoteDescription(answer)
+  ]);
+}
+
+promise_test(async ()=> {
+    const pc1 = new RTCPeerConnection();
+
+    pc1.addTransceiver("audio", { direction: "sendonly" });
+    pc1.addTransceiver("video");
+    pc1.getTransceivers()[0].stop();
+
+    const offer = await pc1.createOffer();
+    assert_false(offer.sdp.includes("m=audio"), "offer should not contain an audio m section");
+}, "An offer should be generated even if there is no valid transceiver");
```

Comment:
Incorrect test name

Reply:
Question: does a stopped m= section not show up in the offer?

Reply:
Not as per my reading of https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-24#section-5.2
Firefox and Safari behaves accordingly there